code/modules/background/space_sectors/coalition.dm

```diff
@@ -0,0 +1,26 @@
+/datum/space_sector/haneunim
+	name = SECTOR_HANEUNIM
+	description = "Centered around the yellow star, Haneunim, the system contains four major celestial bodies, including the planet of Konyang. The whole system bears evidence of ancient mining operations, missing an asteroid belt to separate the inner and outer system with what few rocks remain being mostly picked clean of minerals."
+	skybox_icon = "badlands"
+	possible_exoplanets = list(/obj/effect/overmap/visitable/sector/exoplanet/lava/huozhu, /obj/effect/overmap/visitable/sector/exoplanet/barren/hwanung, /obj/effect/overmap/visitable/sector/exoplanet/barren/qixi, /obj/effect/overmap/visitable/sector/exoplanet/grass/grove/konyang)
+	starlight_color = "#b13636"
+	starlight_power = 2
+	starlight_range = 4
+
+/datum/space_sector/weeping_stars
+	name = SECTOR_WEEPING_STARS
+	description = "The wounds of the Great Interstellar War still have not healed, with planets dotting the Weeping Stars bearing their scars from the massive conflict."
+	skybox_icon = "badlands"
+	possible_exoplanets = list(/obj/effect/overmap/visitable/sector/exoplanet/barren/asteroid, /obj/effect/overmap/visitable/sector/exoplanet/lava, /obj/effect/overmap/visitable/sector/exoplanet/barren, /obj/effect/overmap/visitable/sector/exoplanet/snow, /obj/effect/overmap/visitable/sector/exoplanet/grass/grove, /obj/effect/overmap/visitable/sector/exoplanet/desert)
```

Comment:
I don't really want the snow, grove and desert exoplanets spawning at all until there's actual content for them, so I'll be seeing with the others if this should be excluded from the list here.